### PR TITLE
Updated FrozenLakeEnv docstring

### PR DESCRIPTION
**Pull request by johnMinelli**
Updated FrozenLakeEnv argument descriptions to allow easier creation of a randomly generated map.

### CODE REVIEW

File: gym/envs/toy_text/frozen_lake.py [hunk 1]
```diff
@@ -104,13 +104,21 @@ class FrozenLakeEnv(Env):
...
     `desc`: Used to specify custom map for frozen lake. For example,
 
         desc=["SFFF", "FHFH", "FFFH", "HFFG"].
 
+        A random generated map can be specified by calling the function `generate_random_map`. For example,
+
+        ```
+        from gym.envs.toy_text.frozen_lake import generate_random_map
+
+        gym.make('FrozenLake-v1', desc=generate_random_map())
```
Comment (pseudo-rnd-thoughts) [inline gym/envs/toy_text/frozen_lake.py:119]: Could you specify the map size in `generate_random_map`

Comment (johnMinelli) [reply gym/envs/toy_text/frozen_lake.py]: Sure, it's fine like that?

Comment (pseudo-rnd-thoughts) [reply gym/envs/toy_text/frozen_lake.py]: Yes, that looks good

**Comment by younik** [comment]
related to https://github.com/openai/gym/issues/2866

**Comment by pseudo-rnd-thoughts** [comment]
LGTM

**Pull request merged**